src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -501,6 +540,24 @@ def download_secret(client, vault_base_url, secret_name, file_path, encoding=Non
         raise ex
 
 
+def backup_secret(client, vault_base_url, secret_name, file_path):
+    backup = client.backup_secret(vault_base_url, secret_name).value
+    with open(file_path, 'wb') as output:
+        output.write(backup)
+
+
+backup_secret.__doc__ = KeyVaultClient.backup_secret.__doc__
+
+
+def restore_secret(client, vault_base_url, file_path):
```

Comment:
You should add a scenario tests that backs up and restores a secret successfully. From doing the original KV commands, I would be surprised if more edge cases didn't need to be covered on account of Python2 vs 3 and Linux vs Windows.